chatminer/visualizations.py

```diff
@@ -15,18 +31,21 @@
 from mpl_toolkits.axes_grid1 import make_axes_locatable
 from wordcloud import STOPWORDS, WordCloud
 
+StrSeries = pd.Series[str] if sys.version_info >= (3, 10) else pd.Series
+IntSeries = pd.Series[int] if sys.version_info >= (3, 10) else pd.Series
```

Comment:
Do you have more context on why this is needed?

Reply:
`pd.Series` doesn't support sub-scripting in versions below 3.10.

Reply:
Got you! Working with pandas is tedious sometimes, I wonder if switching to a lib like Polars might make the code cleaner overall